builtin/providers/google/resource_compute_instance.go

```diff
@@ -246,6 +246,20 @@ func resourceComputeInstance() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"automatic_restart": &schema.Schema{
```

Comment:
May as well also add the third one "on_host_maintenance"

They don't have to be ForceNew either, but it's fine to make them ForceNew in a first pass and then add update ability later.